appveyor.yml

```diff
@@ -67,8 +67,7 @@ install:
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
     - cmd: conda config --add channels conda-forge
     - cmd: conda config --set show_channel_urls true
-    - cmd: appveyor-retry conda update --yes --quiet conda
-
+    - cmd: conda update --yes --quiet conda
```

Comment:
Please revert this change to the `appveyor.yml` file.

Reply:
I'll update this to the version in the current master.